src/qibocal/protocols/characterization/utils.py

```diff
@@ -328,51 +331,27 @@ def fit_punchout(data: Data, fit_type: str):
 
     for qubit in qubits:
         qubit_data = data[qubit]
-        freqs = np.unique(qubit_data.freq)
-        nvalues = len(np.unique(qubit_data[fit_type]))
-        nfreq = len(freqs)
-        signals = np.reshape(qubit_data.signal, (nvalues, nfreq))
+        freqs = qubit_data.freq
+        amps = getattr(qubit_data, fit_type)
+        signal = qubit_data["signal"]
```

Comment:
```suggestion
        signal = qubit_data.signal
```
Just for consistency